src/CaptureClient/ApiEventProcessor.cpp

```diff
@@ -117,6 +117,7 @@ void ApiEventProcessor::ProcessStopEventLegacy(const orbit_api::ApiEvent& api_ev
   timer_info.set_group_id(0);
   timer_info.set_address_in_function(0);
 
+  timer_info.set_api_scope_group_id(api_event_id_provider_->ProvideId(timer_info));
```

Comment:
maybe add a CHECK that api_event_id_provider_ exists here and below.

Reply:
Nothing to check anymore.